Trippy/Models/Itinerary.m

```diff
@@ -0,0 +1,35 @@
+//
+//  Itinerary.m
+//  Trippy
+//
+//  Created by Catherine Lu on 7/11/22.
+//
+
+#import "Itinerary.h"
+
+@implementation Itinerary
+
+- (instancetype) initWithDictionary:(NSDictionary *)dict {
+    self = [super init];
+    
+    if (self) {
+        self.directionsJson = dict;
+        NSDictionary *bounds = dict[@"routes"][0][@"bounds"]; // Bounds of routes
```

Comment:
You should be able to pull `dict[@"routes"][[0] out; you reuse it a few times below.